backend/fms_core/tests/test_resources.py

```diff
@@ -170,77 +170,27 @@ def test_sample_extracted_from_version_count(self):
 
     def test_first_sample_extraction_import(self):
         self.load_samples_extractions()
-
         # Test first extraction
         s = Sample.objects.get(container__barcode="tube003")
-        self.assertListEqual(s.extracted_from.volume_history, [
```

Comment:
I would replace this by an assertEqual(s.extracted_from.volume, Decimal("9.000")). This would still test the existence of a lineage and the correct resulting volume of the parent. Try not to remove tests unless they are not valid anymore. We need more tests not less ;).